html/changelogs/N3X15-deus-ex.yml

```diff
@@ -0,0 +1,10 @@
+author: N3X15
+delete-after: true
+changes:
+  - experiment: "Deus Ex Update:"
+  - rscadd: Lungs are now fully modular and actually process gas for humans.  Lungs must be compatible with a species or they will die of suffocation.
+  - rscadd: Vox and Plasmamen have their own lungs.  These alien lungs are not available at organ printers, requiring stockpiling from donor monkey-humans.
```

Comment:
Even if they weren't, there wouldn't be a way of getting plasmamen lungs like this.